bundles/org.eclipse.e4.ui.workbench.renderers.swt/src/org/eclipse/e4/ui/internal/workbench/renderers/swt/AbstractTableInformationControl.java

```diff
@@ -403,6 +403,9 @@ private void setMatcherString(String pattern) {
 			fSearchPattern = null;
 		} else {
 			SearchPattern patternMatcher = new SearchPattern();
+			if (!pattern.startsWith("*")) { //$NON-NLS-1$
+				pattern = "*" + pattern; //$NON-NLS-1$
```

Comment:
Change looks good to me.
But one question: Is this class only used for the Cevron-List? If not we may create unwanted side effects..